build.gradle.kts

```diff
@@ -143,6 +144,24 @@ tasks.create("generate-models-from-openapi-document-print-api-sqs-messaging.yaml
     packageName.set("uk.gov.dluhc.printapi.messaging")
 }
 
+// Codegen the Print Provider schemas from yamlschema into java pojos
+jsonSchema2Pojo {
```

Comment:
This is the same codegen plugin I showed you the other week @Neil-Massey  , but without that nasty hack I had to do re: reversing properties because of a bug with their builder and constructor args order ...
There was a new release on Oct 17th (4.5.0) which seems to have fixed that, which is good, but it also means the plugin is being actively maintained 👍

Reply:
Beast of options... at least they now set what they should :-)